modules/manage/partials/internal-use-property.adoc

```diff
@@ -0,0 +1 @@
+WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by https://support.redpanda.com/hc/en-us[Redpanda Support^].
```

Comment:
So does this appear once at the top of the topic? If so, then:

WARNING: _The properties described here are for Redpanda internal use only_. Do not modify it unless specifically instructed by https://support.redpanda.com/hc/en-us[Redpanda Support^]. Using any property here without explicit guidance from Redpanda support could result in data loss.

Reply:
No. The idea here is to inject this admonition into every property that is meant for internal-only usage. 

Do you want to create a separate section for those properties?

Reply:
Going with individual admonitions for properties after a decision in a Zoom Call.